feat(stock movement report) add currency symbol and round amounts

jeremielodi opened the pull request.

jeremielodi committed:
feat(stock movement report) add currency symbol and round amounts

jniles commented on server/controllers/stock/reports/stock_movements.report.handlebars:
```diff
@@ -76,7 +76,7 @@
                   <td class="text-right">{{quantity}}</td>
                   <td class="text-right">{{unit_cost}}</td>
-                  <td class="text-right">{{multiply quantity unit_cost}}</td>
+                  <td class="text-right">{{currency cost ../metadata.enterprise.currency_id}}</td>
```
I'm almost certain this should be `../../metadata.enterprise.currency_id`.  Have you tried changing the enterprise currency to see if this changes?

jniles replied:
So, it looks like I was wrong - I tested this, and the currency value does not change.  The way you had it before was right.

If I'm wrong in the future, please tell me instead of just blindly changing the code.

jniles commented on server/controllers/stock/reports/stock/movements_report.js:
```diff
@@ -38,7 +39,10 @@ function stockMovementsReport(req, res, next) {
     .then((rows) => {
-      data.rows = rows;
+      data.rows = rows.map(row => {
+        row.cost = util.roundDecimal(row.quantity * row.unit_cost, 3);
```
👍

jniles committed:
fix(stock): respect currency in mvmt report

This commit ensures that the stock movement report respects the
enterprise currency.

mbayopanda commented on server/controllers/stock/reports/stock_movements.report.handlebars:
```diff
@@ -59,41 +57,29 @@
+                <td>{{label}}</td>
+                <td class="text-right">{{quantity}}</td>
+                <td class="text-right">{{currency unit_cost ../../metadata.enterprise.currency_id}}</td>
+                <td class="text-right">{{currency cost ../../metadata.enterprise.currency_id}}</td>
```
👍

mbayopanda reviewed (approved):
LGTM 👍 

bors r+
